### DISCUSSION TITLE
metadata return type has been changed

### DISCUSSION THREAD

**Discussion by linzhidong1984** [dcm4che/dcm4chee-arc-light#2921] [Q&A] [answered]
I use ohif-vewer as front and dcm4chee-arc-ligth-5.22.6 as backend in my project.But ,the ohif-vewer can not work ;
When i use  dcm4chee-arc-ligth-5.22.0,it can works well;
The reason is ,in dcm4chee-arc-ligth-5.22.6 ,
this api http://xxx/dcm4chee-arc/aets/DCM4CHEE/rs/studies/xxxx/series/xxx/metadata  
returns some attributes like this :
00200011":{"vr":"IS","Value":["4"]},"00200012":{"vr":"IS","Value":["2"]},"00200013":{"vr":"IS","Value":["234"]} 
the value of "Value" is String;
in dcm4chee-arc-ligth-5.22.0 returns:
"00200011":{"vr":"IS","Value":[4]},"00200012":{"vr":"IS","Value":[2]},"00200013":{"vr":"IS","Value":[1]}
the value of "Value" is Int;

can someone help me to solve this problem? thanks a lot.

**Comment by gunterze** [comment, marked as answer]
JSON type for encoding IS & DS values has been changed from JSON Number to JSON String according CP 1861 `ftp://medical.nema.org/medical/dicom/final/cp1861_ft_changeJSONvaluetypemappingISandDS.pdf` in 5.22.6 (https://github.com/dcm4che/dcm4che/issues/803) to avoid loose of invalid IS & DS values in objects received in binary or XML format on export as JSON.

Seems ohif-vewer is not compliant to CP 1861.

Opened #2922

**Comment by Mamta92** [comment]
Hi @linzhidong1984  @gunterze 
I'm facing a 409 error conflict when uploading images. Can you check?

More details here - 
https://github.com/dcm4che/dcm4chee-arc-light/discussions/4258
